PhpAmqpLib/Helper/DebugHelper.php

```diff
@@ -22,6 +22,10 @@ class DebugHelper
      * @param string $PROTOCOL_CONSTANTS_CLASS
      */
     public function __construct($PROTOCOL_CONSTANTS_CLASS) {
+        if(!defined('STDIN'))  define('STDIN',  fopen('php://stdin',  'r'));
```

Comment:
Do we need `STDIN` here? It's not opened by default in some environments for a reason.
We don't need `STDERR` too, but it's not dangerous.